aws_lambda_powertools/utilities/feature_flags/schema.py

```diff
@@ -31,6 +32,27 @@ class RuleAction(str, Enum):
     KEY_NOT_IN_VALUE = "KEY_NOT_IN_VALUE"
     VALUE_IN_KEY = "VALUE_IN_KEY"
     VALUE_NOT_IN_KEY = "VALUE_NOT_IN_KEY"
+    SCHEDULE_BETWEEN_TIME_RANGE = "SCHEDULE_BETWEEN_TIME_RANGE"  # hour:min 24 hours clock UTC time
+    SCHEDULE_BETWEEN_DATETIME_RANGE = "SCHEDULE_BETWEEN_DATETIME_RANGE"  # full datetime format
+    SCHEDULE_BETWEEN_DAYS_OF_WEEK = "SCHEDULE_BETWEEN_DAYS_OF_WEEK"
+
+
+class TimeKeys(Enum):
+    CURRENT_TIME_UTC = "CURRENT_TIME_UTC"
+    CURRENT_DAY_UTC = "CURRENT_DAY_UTC"
```

Comment:
One more that I forgot. Otherwise I would think these are numbers (1-31)

```suggestion
    CURRENT_DAY_OF_WEEK_UTC = "CURRENT_DAY_OF_WEEK_UTC"
```

Reply:
fixed